pep-0659.rst

```diff
@@ -149,14 +145,14 @@ Each instruction that would benefit from specialization is replaced by an
 adaptive version during quickening. For example,
 the ``LOAD_ATTR`` instruction would be replaced with ``LOAD_ATTR_ADAPTIVE``.
 
-Each adaptive instruction maintains a counter,
-and periodically attempts to specialize itself.
+Each adaptive instruction maintains a counter, and periodically attempts
+to specialize itself.
```

Comment:
There is an almost-identical sentence two paragraphs down. Maybe remove one of them?